File: Javascript/Stack.js
class Stack{
    /* Stack constructor*/
    constructor(){
        this.stack = [];
        this.size = 0;
    }

    /* size()
     * return size of the stack
     */
    size(){
        return this.size;
    }

    /* empty()
     * return True if stack is empty, False otherwise
     */
    empty(){
        return this.size == 0;
    }

    /* push(x)
     * push [x] into stack
     */
    push(x){
        this.stack.push(x);
        this.size++;
    }

    /* pop()
     * remove and return top tiem from queue
     */
    pop(){
        var data = this.stack[this.size - 1];
        this.stack = this.stack.slice(0, -1);
        this.size--;
        return data;
    }

    /* print current queue as a list */
    print(){
        var rev = [];
        for (var i = this.stack.length - 1; i >= 0; i--){
            rev.push(this.stack[i]);
        }
        console.log(rev);
    }
}

var stk = new Stack();
process.stdout.write("Current Stack is: Top -> ");
stk.print();

console.log("Push -> 19");
console.log("Push -> 46");
console.log("Push -> 87");
console.log("Push -> 98");
console.log("Push -> 9");
console.log("Push -> 70");
console.log("Push -> 10");
stk.push(19);
stk.push(46);
stk.push(87);
stk.push(98);
stk.push(9);
stk.push(70);
stk.push(10);
process.stdout.write("Current Stack is: Top -> ");
stk.print();

process.stdout.write("The size of current stack is ");
console.log(stk.size());

process.stdout.write("Current Stack is empty: ");
console.log(stk.empty());

process.stdout.write("The popped item from stack is ");
console.log(stk.pop());
process.stdout.write("Current Stack is: Top -> ");
stk.print();

File: Python/Stack.py
class Stack:
    # Stack constructor
    def __init(self):
        self.stack = []
        self.size = int(0)

    # size()
    # return size of the stack
    def size(self):
        return self.size

    # empty()
    # return True if stack is empty, False otherwise
    def empty(self):
        return self.size == 0

    # push(x)
    # push [x] into queue
    def push(self, x):
        self.stack.append(x)
        self.size += 1

    # pop()
    # remove and return top item from queue
    def pop(self):
        data = self.stack[-1]
        self.stack = self.stack[:-1]
        self.size -= 1
        return data

    # print current queue as a list
    def print(self):
        print(self.stack[::-1])

stk = Stack()
print("Current Stack is: Top -> ", end = "")
stk.print()

print("Push -> 19")
print("Push -> 46")
print("Push -> 87")
print("Push -> 98")
print("Push -> 9")
print("Push -> 70")
print("Push -> 10")
stk.push(19)
stk.push(46)
stk.push(87)
stk.push(98)
stk.push(9)
stk.push(70)
stk.push(10)
print("Current Stack is: Top -> ", end = "")
stk.print()

print("The size of current stack is ", end = "")
print(stk.size)

print("Current Stack is empty: ", end = "")
print(stk.empty())

print("The popped item from stack is ", end = "")
print(stk.pop())
print("Current Stack is: Top -> ", end = "")
stk.print()